docs/installation/ccloud-ksql.rst

```diff
@@ -0,0 +1,166 @@
+.. _configuring-ksql:
```

Comment:
The filename is `ccloud-ksql.rst` but this section looks more like a general "Configuring KSQL" document?

Also, we have decided on a different structure for the (general) Configuring KSQL section.

Reply:
Yes, this doesn't have any of the settings we actually need to to configure ccloud. In particular, the following needs to be added to the `/etc/ksql/ksql-server.properties` file in order to connect with CCLoud servers:

```
bootstrap.servers=<a comma separated list of the the ccloud broker endpoints. eg. r0.great-app.confluent.aws.prod.cloud:9092,r1.great-app.confluent.aws.prod.cloud:9093,r2.great-app.confluent.aws.prod.cloud:9094>
ksql.sink.replicas=3
replication.factor=3
security.protocol=SASL_SSL
sasl.mechanism=PLAIN
sasl.jaas.config=\
    org.apache.kafka.common.security.plain.PlainLoginModule required \
    username="<confluent cloud access key>" \
    password="<confluent cloud secret>";
```

Reply:
@apurvam @miguno yes this is a commit mistake - will fix now.

Reply:
fixed https://github.com/confluentinc/ksql/pull/955/commits/e87d06390785994c1b69b6dcaf1974fc755735b4